kotlin-audio/src/main/java/com/doublesymmetry/kotlinaudio/notification/NotificationManager.kt

```diff
@@ -245,10 +251,12 @@ class NotificationManager internal constructor(private val context: Context, pri
                 }
 
                 setMediaSessionToken(mediaSession.sessionToken)
+                setPlayer(exoPlayer)
             }
         }
     }
 
+    // FIXME: This functions seems wrong. It does not do what the name suggests...
     fun clearNotification() {
         mediaSession.isActive = false
         internalManager?.setPlayer(null)
```

Comment:
Not sure what is the point of this function and why it was added. It's called `clearNotification` but it does something totally unrelated. Plus, the only intended way to clear our foreground notification is from our `MusicService`, since it created it.